src/Cache/Engine/RedisEngine.php

```diff
@@ -82,6 +82,10 @@ public function init(array $config = [])
             return false;
         }
 
+        if (!empty($config['host'])) {
+            $this->config('server', $config['host']);
```

Comment:
Would it make more sense to set the array key directly instead of calling the function?